webapp/tests/elster_client/test_elster_client.py

```diff
@@ -520,6 +518,38 @@ def tearDown(self):
             MockErica.available_idnrs.remove(self.new_idnr)
 
 
+class TestValidateTaxNumber:
+
+    def test_if_tax_number_is_valid_then_return_true(self):
+        state_abbreviation = 'BY'
+        tax_number = '19811310010'
+        result = validate_tax_number(state_abbreviation, tax_number)
+
+        assert result is True
+
+    def test_if_tax_number_is_invalid_then_return_false(self):
+        state_abbreviation = 'BY'
+        tax_number = '00000111111'
+        MockErica.tax_number_is_invalid = True
+        try:
+            result = validate_tax_number(state_abbreviation, tax_number)
+
+        finally:
+            MockErica.tax_number_is_invalid = False
+
+        assert result is False
```

Comment:
Not sure how much sense this makes, just a suggestion: Should we add a test to test what happens if the Bufa_nr is invlaid? No, because we already cover this in erica, correct?